client/post-editor/media-modal/gallery-help.jsx

```diff
@@ -80,7 +80,7 @@ const EditorMediaModalGalleryHelp =  React.createClass( {
 				context={ renderContext }
 				position="bottom"
 				isVisible={ ! isMobile() }
-				className="popover is-dialog-visible">
+				className="popover popover__gallery-help is-dialog-visible">
```

Comment:
I notice you're adding a new class here that doesn't seem related to the other changes in this commit. Just checking, is this something we need?